configs/milvus.yaml

```diff
@@ -277,7 +277,7 @@ common:
   # Valid values: [auto, avx512, avx2, avx, sse4_2]
   # This configuration is only used by querynode and indexnode, it selects CPU instruction set for Searching and Index-building.
   simdType: auto
-  indexSliceSize: 4 # MB
+  indexSliceSize: 16 # MB
```

Comment:
Maybe `16` is a little larger. After codec, the binlog corresponding to an index file with `16` MB will be larger, which is not recommended by `S3`.

Reply:
The S3 limit is < 1GB, and object > 100MB is recommended to use multi part load.
So far I believe 16MB should be ok but it shouldn't be anything larger than that.